minions.md

```diff
@@ -144,6 +146,22 @@ Agility rewards you with marks of grace which you can use to buy Graceful equipm
 
 Click [here](https://i.imgur.com/LZGlSgj.png) for **Agility** XP rates and **Marks of Grace** per hour.
 
+#### Monkey Backpacks
+
+When completing laps of the Ape Atoll agility course you can earn a [monkey backpack](https://oldschool.runescape.wiki/w/Monkey_(Monkey_Madness_II))! These are obtained automatically whenever you reach the lap count required for each backpack.
```

Comment:
this link doesnt look like it works

Reply:
edited idk why it should have been a direct copy of the url. In any case it now links to the wiki page for the ape atoll course generally which has them listed too.